### PR TITLE
Add debug documentation and assets

### PR DESCRIPTION
**Pull request by radu-matei**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "configurations": [
+        {
+            "name": "Debug Signy",
+            "type": "go",
+            "request": "launch",
+            "mode": "exec",
+            "program": "${workspaceFolder}/bin/signy-debug",
+            "args": [
+                "sign",
+                "/home/radu/projects/src/github.com/cnabio/signy/testdata/cnab/bundle.json",
```
Comment (radu-matei) [inline .vscode/launch.json:15]: TODO
Replace this with `${workspaceFolder}`

Comment (trishankatdatadog) [reply .vscode/launch.json:15]: This can go into your local .gitignore, no?

Comment (radu-matei) [reply .vscode/launch.json:15]: Sure - but the intention was to add a document describing how to debug Signy.

**Comment by radu-matei** [comment]
This might be better suited for a documentation section only.

**Pull request closed without merging**